node/lib/util/merge_util.js

```diff
@@ -98,17 +98,23 @@ const checkForMerge = co.wrap(function *(path) {
  * @param {Object} conflicts map from name to commit causing conflict
  * @return {String} conflict message
  */
-exports.formatConflictsMessage = function(conflicts) {
+const getBareMergeConflictsMessage = function(conflicts) {
     if (0 === Object.keys(conflicts).length) {
         return "";
     }
     let errorMessage = "CONFLICT (content): \n";
     const names = Object.keys(conflicts).sort();
     for (let name of names) {
-        errorMessage += `Conflicting entries for submodule: ` + 
-            `${colors.red(name)}\n`;
+        const conflict = conflicts[name];
```

Comment:
Can you break this into two commits -- one for the conflicts change and the other for the author stuff?